Dockerfile

```diff
@@ -45,7 +45,6 @@ COPY . /code/
 # copy built react and node libraries for production mode
 COPY --from=node-prod-deps /usr/src/app/package-lock.json /code/package-lock.json
 COPY --from=node-prod-deps /usr/src/app/webpack-stats.json /code/webpack-stats.json
-COPY --from=node-prod-deps /usr/src/app/assets/dist/static /code/static
```

Comment:
I had to remove the line since (`import 'typeface-roboto'`)  the roboto font now is not imported anymore with this [change](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/535/commits/f5318bb7588a45cf06aa5f1d05d414506b8b5e82)